dotnet/src/SemanticKernel.Core/Configuration/KernelConfig.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.SemanticKernel;
+
+public class KernelConfig
```

Comment:
Not sure what is the purpose of this class, but our goal would be to refactor only extension methods for `IKernelBuilder` and `IServiceCollection` `Add...` methods.

Reply:
So far, we didn't see a requirement for such class. And without usage examples it's hard to understand the benefits. My recommendation would be to stick to the original issue that mentioned only updates in `Add...` extension methods. It will simplify the review and merging of this PR. 

If you feel that such class will provide some benefits, you can create new issue with all the details, including usage examples, and we can handle it separately. Hope that makes sense!

Reply:
I have deleted the class.